servicetalk-loadbalancer-experimental/src/main/java/io/servicetalk/loadbalancer/LoadBalancingPolicy.java

```diff
@@ -15,34 +15,14 @@
  */
 package io.servicetalk.loadbalancer;
 
-import io.servicetalk.client.api.LoadBalancedConnection;
-
-import java.util.List;
-
 /**
  * Definition of the selector mechanism used for load balancing.
- * @param <ResolvedAddress> the type of the resolved address
- * @param <C> the type of the load balanced connection
  */
-public interface LoadBalancingPolicy<ResolvedAddress, C extends LoadBalancedConnection> {
-
-    /**
-     * The default fail-open policy to use for {@link HostSelector} implementations.
-     */
-    boolean DEFAULT_FAIL_OPEN_POLICY = false;
+public interface LoadBalancingPolicy {
```

Comment:
Note that by dropping the type parameters we 'lock' in the notion that we can load balance over any type of LoadBalancedConnection. That seems like a reasonable assumption for the non-power use case.

Reply:
It's a bit weird that `LoadBalancingPolicy` doesn't have any methods except just `name()`. How users should understand this interface?

Reply:
Ideally it isn't an open interface but something akin a sealed an interface. I suppose we can do that by making it an abstract class with a package private constructor.